main.gms

```diff
@@ -1143,6 +1143,14 @@ parameter
 ;
   c_changeProdCost = 5;  !! def = 5
 *'
+parameter
+  cm_LearningSpillover      "Activate Learningspillover from foreign capacity in learning technogolies"
+;
+  cm_LearningSpillover = 1; !! def 1 = Learningspillover activated (set to 0 to deactivate)
+*'
+*' * if Learningspillover is deactivated, foreign capacity is set to the level of 2020 in technology learning.
```

Comment:
I would add something like: "This means that in the model, each region's learning depends on its OWN additional capacity investment after 2020 in comparison to the GLOBAL cumulative capacity until 2020, so for small regions learning is very slow. This is a very pessimistic interpretation of 'no learning spillovers', as every region has to follow the global learning curve"

Reply:
with "follow the global learning curve" I mean that in `cm_learningSpillover = 0` a country only manages to "double capacity" (which is the moment that the costs are reduced by the learning rate value) when its own capacity addition is equal to the GLOBAL capacity in 2020. 

so I think my initial formulation can be misunderstood - maybe "as every region has to climb up the global learning curve all by itself" is better?

Reply:
Great, thank you for the further explanation. We have added your suggestion in main.gms, see last commit.